src/bootstrap/src/core/metadata.rs

```diff
@@ -89,12 +86,8 @@ fn workspace_members(build: &Build) -> impl Iterator<Item = Package> {
 
     // Collects `metadata.packages` from all workspaces.
     let packages = collect_metadata("Cargo.toml");
-    let cargo_packages = collect_metadata("src/tools/cargo/Cargo.toml");
     let ra_packages = collect_metadata("src/tools/rust-analyzer/Cargo.toml");
```

Comment:
While we're at it, shouldn't we also remove Rust analyzer? It also isn't a member of the workspace, right?